src/decisionengine/framework/engine/tests/fixtures.py

```diff
@@ -1,22 +1,25 @@
 '''pytest defaults'''
-import random
-import string
 import threading
 
-import psycopg2
 import pytest
-from pytest_postgresql.janitor import DatabaseJanitor
 
 import decisionengine.framework.engine.de_client as de_client
 import decisionengine.framework.engine.de_query_tool as de_query_tool
 
-from decisionengine.framework.dataspace.datasources.tests.fixtures import DE_DB_HOST, DE_DB_USER, DE_DB_PASS, DE_DB_NAME, DE_SCHEMA, PG_PROG, DE_DB
-from decisionengine.framework.engine.DecisionEngine import _get_de_conf_manager, _create_de_server, parse_program_options
+from decisionengine.framework.dataspace.datasources.tests.fixtures import (
+    PG_DE_DB_WITH_SCHEMA,
+    PG_PROG,
+    DATABASES_TO_TEST,
+)
+from decisionengine.framework.engine.DecisionEngine import (
+    _get_de_conf_manager,
+    _create_de_server,
+    parse_program_options,
+)
 from decisionengine.framework.util.sockets import get_random_port
 from decisionengine.framework.taskmanager.TaskManager import State
 
-__all__ = ['DE_DB_HOST', 'DE_DB_USER', 'DE_DB_PASS', 'DE_DB_NAME', 'DE_SCHEMA',
-           'PG_PROG', 'DE_DB', 'DE_HOST', 'DEServer']
+__all__ = ["DATABASES_TO_TEST", "PG_DE_DB_WITH_SCHEMA", "PG_PROG", "DEServer"]
```

Comment:
DE_HOST was removed from __all__ because it is in DEServer and no need to use it directly?

Reply:
Yep, and it wasn't used elsewhere, so the `__all__` export more clearly matches what is consumed from this file.